pkg/checker/cluster_checker.go

```diff
@@ -96,6 +99,13 @@ func (n *ClusterChecker) Check(cluster *v2.Cluster, phase string) error {
 	return n.Output(NodeList)
 }
 
+func isControlPlaneNode(node corev1.Node) bool {
+	if _, ok := node.Labels["node-role.kubernetes.io/control-plane"]; ok {
```

Comment:
Issues with older versions need to be considered

Reply:
Agreed,I'll add an old version label to check.